### PR TITLE
cc-104 multiple galleries per public site

### PR DESCRIPTION
**Pull request by sethphillips**

### CODE REVIEW

File: app/routes/gallery.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    return Ember.RSVP.hash({
+      gallery,
+      shows,
+    });
+  },
+
+  setupController(controller, model){
+		this._super(...arguments);
+    controller.set('model',model);
+		window.scrollTo(0,0);
```
Comment (raytiley) [inline app/routes/gallery.js]: this for sure should be guarded for fastboot.

File: tests/integration/components/paginated-show-list-test.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import { moduleForComponent, test } from 'ember-qunit';
```
Comment (raytiley) [inline tests/integration/components/paginated-show-list-test.js]: YAY TESTS! ❤️

**Comment by raytiley** [comment]
- remove unneeded wonky spinner
 - remove unused tests
- fix scroll fastboot position

**Comment by sethphillips** [comment]
@raytiley 🌮

**Comment by raytiley** [comment]
@sethphillips sorry... rebase please.

**Pull request merged**